test/files/neg/unused.check

```diff
@@ -0,0 +1,21 @@
+unused.scala:7: warning: pattern var a in value patvars is never used: use a wildcard `_` or suppress this warning with `a@_`
+    case C(a, _, _, _) => 1
+           ^
+unused.scala:8: warning: pattern var b in value patvars is never used: use a wildcard `_` or suppress this warning with `b@_`
+    case C(_, b, c@_, _) => 2
+              ^
+unused.scala:9: warning: pattern var a in value patvars is never used: use a wildcard `_` or suppress this warning with `a@_`
+    case C(a, b@_, c, d@_) => 3
+           ^
+unused.scala:9: warning: pattern var c in value patvars is never used: use a wildcard `_` or suppress this warning with `c@_`
+    case C(a, b@_, c, d@_) => 3
+                   ^
+unused.scala:10: warning: pattern var e in value patvars is never used: use a wildcard `_` or suppress this warning with `e@_`
+    case e => 4
+         ^
+unused.scala:8: warning: unreachable code
+    case C(_, b, c@_, _) => 2
+                            ^
```

Comment:
I wonder why this is the only unreachability warning.

Reply:
Probably at most one unreachable warning per match expression?